static/src/javascripts/projects/common/modules/support/articleCount.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { storage } from '@guardian/libs';
+import { incrementDailyArticleCount } from 'common/modules/support/articleCount';
+
+jest.mock('raven-js', () => ({
+	config() {
+		return this;
+	},
+	install() {
+		return this;
+	},
+	captureException: jest.fn(),
+}));
+
+const today = Math.floor(Date.now() / 86400000); // 1 day in ms
+
+describe('history', () => {
+	afterEach(() => {
+		storage.local.remove('gu.history.dailyArticleCount');
+	});
+
+	it('creates daily history if empty', () => {
+		incrementDailyArticleCount();
+
+		expect(storage.local.get('gu.history.dailyArticleCount')).toEqual([
+			{ day: today, count: 1 },
+		]);
+	});
+
+	it('removes old daily history while incrementing the article count', () => {
+		const counts = [{ day: today - 70, count: 9 }];
+		storage.local.set('gu.history.dailyArticleCount', counts);
+
+		incrementDailyArticleCount();
+
+		expect(storage.local.get('gu.history.dailyArticleCount')).toEqual([
+			{ day: today, count: 1 },
+		]);
+	});
```

Comment:
Maybe not necessary - but we don't have the standard increment on the same day test?

Reply:
👍 Just pushed a commit with an extra test